include/micm/solver/rosenbrock.hpp

```diff
@@ -585,14 +587,14 @@ namespace micm
 
    protected:
     /// @brief Computes the scaled norm of the vector errors
-    /// @param original_number_densities the original number densities
-    /// @param new_number_densities the new number densities
+    /// @param Y the original vector
+    /// @param new_number_densities the new vector
     /// @param errors The computed errors
     /// @return
     double NormalizedError(
-        MatrixPolicy<double> original_number_densities,
-        MatrixPolicy<double> new_number_densities,
-        MatrixPolicy<double> errors);
+      const MatrixPolicy<double>& Y,
+      const MatrixPolicy<double>& Ynew,
```

Comment:
if we're changing the names, we should probably follow our style of:
```suggestion
      const MatrixPolicy<double>& y,
      const MatrixPolicy<double>& y_new,
```

Reply:
or we can do this with #129